python/module/openstudio.py

```diff
@@ -28,6 +28,7 @@
     from . import openstudioosversion as osversion
     from . import openstudioradiance as radiance
     from . import openstudiosdd as sdd
+    from . import openstudioualfalfa as alfalfa
```

Comment:
what's that extra "u" in there? I think you meant `openstudioalfalfa` @TShapinsky